assets/src/setup/components/options-context-provider.js

```diff
@@ -64,6 +100,16 @@ export function OptionsContextProvider( { children, optionsRestEndpoint } ) {
 				delete updatesToSave.reader_theme;
 			}
 
+			// If this is the first time running the wizard and mobile_redirect is not in updates, set mobile_redirect to true.
+			// We do this here instead of in the fetch effect to prevent the exit confirmation before the user has interacted.
+			if ( ! originalOptions.wizard_completed && ! ( 'mobile_redirect' in updatesToSave ) ) {
+				updatesToSave.mobile_redirect = true;
```

Comment:
I think it would be better to set it to the original value:

```suggestion
				updatesToSave.mobile_redirect = originalOptions.mobile_redirect;
```

Or what would be better is to merge the default options with the updated ones here, so that no other values are missed. Something like:

```js
updatesToSave = { ...originalOptions, ...updates };
```

Reply:
I think the suggested change works best, because there's no need to send back _all_ the options, just the changed ones.